docs/en/deploy/Running-Alluxio-On-Yarn.md

```diff
@@ -57,10 +59,6 @@ $ ${HADOOP_HOME}/sbin/start-yarn.sh
 
 ## Alluxio YARN integration
```

Comment:
maybe calling this "Running Alluxio as YARN application"? `Alluxio YARN integration` is not very informative in doc like this.

Reply:
updated